src/components/card-health.js

```diff
@@ -37,6 +38,19 @@ export class Card {
       if(this.repo.vscan && this.repo.vscan.infected_files) {
         this.infected_files = this.repo.vscan.infected_files;
       }
+
+      if (this.repo.badges && this.repo.badges.status) {
+        switch(this.repo.badges.status.toLowerCase()) {
+          case 'active':
+            this.badge_status_image = '/img/active_flame_final_28w_35h.svg';
+            break;
+          case 'inactive':
+            this.badge_status_image = '/img/inactive_zzz_final_32w_35h.svg';
+            break;
+          default:
+            this.badge_status_image = null;
```

Comment:
This should be changed so that the default image is 'img/pending_review_final_29w_35h.svg'.
This image is to be used when a repo has not been reviewed yet.

Reply:
You are right... code was added in next commit.